### PR TITLE
avoid punctuation in some links

### PR DESCRIPTION
**Pull request by Haroenv**
For example on npm the link would be made with the colon in there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,6 @@ Build the project and documentation:
 
     $ bundle exec jekyll build
 
-Or serve it on a local server on http://localhost:7998/Font-Awesome/:
+Or serve it on a local server on >http://localhost:7998/Font-Awesome/>:
```
Comment (tagliala) [inline README.md]: is `>http://localhost:7998/Font-Awesome/>` right?

Maybe it should be `<http://localhost:7998/Font-Awesome/>`

but we really don't want a link to localhost, maybe `` `http://localhost:7998/Font-Awesome` `` is better

**Comment by tagliala** [comment]
Thanks, but this file is generated, please fix the proper readme: https://github.com/FortAwesome/Font-Awesome/blob/master/src/README.md-nobuild

Please also take a look at my comment

File: README.md (hunk 1 above)
Comment (Haroenv) [inline README.md]: Yeah, that's a typo. But I think a link to local host still would be beneficial

Comment (tagliala) [inline README.md]: I'm strongly against broken links in the docs

**Comment by Haroenv** [comment]
I'll take a look at using the other file later today

**Comment by Haroenv** [comment]
Solved by #11502

**Pull request closed without merging**